discount/src/main/java/com/kyriba/conference/discount/api/DiscountController.java

```diff
@@ -0,0 +1,60 @@
+package com.kyriba.conference.discount.api;
+
+import com.kyriba.conference.discount.domain.DiscountType;
+import com.kyriba.conference.discount.domain.dto.DiscountDto;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static com.kyriba.conference.discount.domain.DiscountType.*;
+import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8_VALUE;
+
+/**
+ * @author Igor Lizura
+ */
+@RestController
+@AllArgsConstructor
+@RequestMapping("/v1/discount")
+public class DiscountController {
+    @ResponseBody
+    @GetMapping(produces = APPLICATION_JSON_UTF8_VALUE)
```

Comment:
produces and consumes may be configured at class level

Reply:
done